.gitignore

```diff
@@ -37,6 +37,7 @@ CMakeLists.txt.user
 scan-build
 .deps
 .libs
+.idea
```

Comment:
Purely out of curiosity, what is this directory associated with?

Reply:
IntelliJ. :-)